kalite/topic_tools/__init__.py

```diff
@@ -165,39 +201,43 @@ def create_thumbnail_url(thumbnail):
 
 CONTENT          = None
 CACHE_VARS.append("CONTENT")
-def get_content_cache(force=False, annotate=False):
+def get_content_cache(force=False, annotate=False, language="en"):
     global CONTENT, CONTENT_FILEPATH
```

Comment:
Following the pattern above, content_cache is now generated on a per language basis.